src/cargo/ops/vendor.rs

```diff
@@ -220,7 +220,7 @@ fn sync(
         let pathsource = PathSource::new(src, id.source_id(), gctx);
```

Comment:
this is targeting `rust-1.80.0`.  Did that as a placeholder until we have `rust-1.80.2`

Reply:
That branch is for `1.80` and would never change as far as I can tell.